patches/components-content_settings-core-browser-content_settings_utils.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/content_settings/core/browser/content_settings_utils.cc b/components/content_settings/core/browser/content_settings_utils.cc
+index dee7eac903da92f2628e5055356f48c23e474ef6..5546bf332c0f5273b5b7e4073bbbb793a7da7f42 100644
+--- a/components/content_settings/core/browser/content_settings_utils.cc
++++ b/components/content_settings/core/browser/content_settings_utils.cc
+@@ -184,6 +184,7 @@ bool CanTrackLastVisit(ContentSettingsType type) {
+   // a valid "initial default" value.
+   if (type == ContentSettingsType::PROTOCOL_HANDLERS)
+     return false;
++  BRAVE_CAN_TRACK_LAST_VISIT
```

Comment:
do we even want to use `CanBeAutoRevoked`? cc @goodov

Reply:
> do we even want to use `CanBeAutoRevoked`? cc @goodov

this a part of an upstream feature which cleans unused permissions after ~60 days. We decided to keep it:
https://bravesoftware.slack.com/archives/C8MP8ME4C/p1667412118906529?thread_ts=1667332958.013179&cid=C8MP8ME4C